.github/workflows/tflint.yml

```diff
@@ -0,0 +1,42 @@
+name: TFLint
+
+on:
+  push:
+    branches: [ "master", "main" ]
+  pull_request:
+    branches: [ "master", "main" ]
+
+jobs:
+  tflint:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Is this necessary? What is OS-specific about running tflint?

Reply:
I'd say it isn't, maybe just use ubuntu-latest rather.
I guess it's to fit any type of GitHub runner available.